software/contrib/sequential_switch.md

```diff
@@ -0,0 +1,42 @@
+# Sequential Switch
+
+This script acts as a sequential switch, routing a copy of the analogue input
```

Comment:
> a copy of the analogue input

[required] We should probably add a note somewhere in this document to set user expectations of the copy. I haven't had a chance to actually run this code, but my expectations are that constant voltages will be a little noisy, smoothly changing voltages will be 'bit-crushed' to some extent, and audio will, i don't know, be totally destroyed?

Reply:
I've added a section on limitations, and added a callout to that section at the top of the readme. Hopefully that covers the bases for the documentation?